packages/connect/src/device/DeviceCommands.ts

```diff
@@ -429,6 +429,9 @@ export class DeviceCommands {
 
     async _commonCall(type: MessageKey, msg?: DefaultMessageResponse['message']) {
         const resp = await this.call(type, msg);
+        if (this.disposed) {
+            throw ERRORS.TypedError('Runtime', 'typedCall: DeviceCommands already disposed');
+        }
```

Comment:
this should be fixing racecondtion in override method 
there is similar code on line 415. but, in case you cancelled called when connect was already behind 415 line handling some flow using recursive calls (see filterCommonTypes), it could have happend that it issued another call to transport layer when the call should have been already stopped.